docs/Makefile

```diff
@@ -51,8 +51,3 @@ env:
 	source env/bin/activate; conda env update --name base --file ./environment.yml
 
 .PHONY: env
-
-# Catch-all target: route all unknown targets to Sphinx using the new
-# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile links
```

Comment:
You still want to keep this, just remove the `Makefile` and `links` as dependencies.